### ISSUE TITLE
PF4: Radio required properties

### ISSUE THREAD

**Issue by dlabrecq** [patternfly/patternfly-react#803]
I installed the latest @patternfly/react-core v1.25.1 package and encountered the following TypeError. It appears that the radio button ID, name, and aria-label are required properties?

![screen shot 2018-10-17 at 3 19 09 pm](https://user-images.githubusercontent.com/17481322/47111073-8a8bbc00-d220-11e8-970c-949668561069.png)

I can understand adding ID/name, but shouldn't aria-label (et al.) be optional?

**Comment by boaz1337** [comment]
It looks like @rvsia patch https://github.com/patternfly/patternfly-react/commit/a62ee271b08b823b3631d38aa1f4579f768894ae#diff-5aebc3b1ceb9e12e2593314778796112R18 changed it to required.

**Comment by dlabrecq** [comment]
Yes, aria-label is now required. Although, I don't believe it should be -- it's not required on all pf4 components. The components should make it easy for apps to implement a11y, but not force it.

Also, ID and name may be required for a radiobutton group, but not for a single radio button.

**Comment by rvsia** [comment]
Hello everyone,

this depends on our vision for the PF4 React. Since the checking of children props has been removed (https://github.com/patternfly/patternfly-react/pull/823 and see discussion here https://github.com/patternfly/patternfly-react/issues/799) we can not alert consumers if children of formGroups do not have labels, ids, etc. So, the simplest solution is to have this props required. Or not to control it and let consumers manage it themselves, but as I said, that depends on what we want to do with a11y.

In terms of a single radio button, I can not imagine use cases for using single radio buttons. There are checkbox or switch components which suit more.

**Comment by tlabaj** [comment]
closed by #930
